### PR TITLE
Update web config file with new scale unit id based on user config changes

### PR DESCRIPTION
**Pull request by Omaaarz**
Update web config file with new scale unit id based on user config changes

### CODE REVIEW

File: src/CLI/SetupToolsOptions/UpdateScaleUnitId.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+            sortedScaleUnits = Config.NonHubScaleUnitInstances();
+            sortedScaleUnits.Sort();
+
+            foreach (ScaleUnitInstance scaleUnit in sortedScaleUnits)
+            {
+                options.Add(new CLIOption() { Name = scaleUnit.PrintableName(), Command = RunUpdateScaleunitId });
+            }
+
+            var screen = new CLIScreen(options, selectionHistory, "Environments:\n", "\nWhich environment would you like to update its scale unit id?: ");
```
Comment (AndreasHassing) [inline src/CLI/SetupToolsOptions/UpdateScaleUnitId.cs:26]:
```suggestion
            var screen = new CLIScreen(options, selectionHistory, "Environments:\n", "\nWhich environment would you like to update the scale unit id of?: ");
```

**Review by AndreasHassing — APPROVED** [review]
Minor comment, LGTM.

**Comment by AndreasHassing** [comment]
Is this for restore? Does DbSync need to run after changing scale unit ID? Not sure I understand exactly why or when this feature is needed.

**Comment by Omaaarz** [comment]
> Is this for restore? Does DbSync need to run after changing scale unit ID? Not sure I understand exactly why or when this feature is needed.

@AndreasHassing Yes, this is for restore. This step will be done before even importing the DB and that's why we shouldn't be doing DBsync at this step. Please see [here](https://microsoft-my.sharepoint.com/:p:/p/omeissaa/EY2hBbLmMipGr2rVVqjlj6YB6ZSYKfBqkHEWQc6AY6FA4A?e=bLuHy1) the scenario for UAT to Devbox(Both CnE).

Importing the DB should be followed by DBsync. 

Please let me know if you have any concerns.

**Comment by AndreasHassing** [comment]
> > Is this for restore? Does DbSync need to run after changing scale unit ID? Not sure I understand exactly why or when this feature is needed.
> 
> @AndreasHassing Yes, this is for restore. This step will be done before even importing the DB and that's why we shouldn't be doing DBsync at this step.

I'm wondering if it makes sense to have this action from the tool stand-alone, or if it should be an integrated part of restore? You may have a design in mind that puts my concerns to rest, if so feel free to merge this 😊.

**Comment by Omaaarz** [comment]
> > > Is this for restore? Does DbSync need to run after changing scale unit ID? Not sure I understand exactly why or when this feature is needed.
> > 
> > 
> > @AndreasHassing Yes, this is for restore. This step will be done before even importing the DB and that's why we shouldn't be doing DBsync at this step.
> 
> @Omaaarz: I'm wondering if it makes sense to have this action from the tool stand-alone, or if it should be an integrated part of restore? You may have a design in mind that puts my concerns to rest, if so feel free to merge this 😊.

I'm trying to have the experience here as close as possible to steps needed in SUM for (Prod -> UAT) operations. That's why I'm keeping it as a standalone step now.

**Pull request merged**